api/database/schema_create.sql

```diff
@@ -0,0 +1,23 @@
+CREATE DATABASE  IF NOT EXISTS `protectplayanow2` /*!40100 DEFAULT CHARACTER SET utf8 */;
+USE `protectplayanow2`;
+
+DROP TABLE IF EXISTS `reading`;
+CREATE TABLE `reading` (
+  `instant` datetime NOT NULL,
+  `deviceId` varchar(36) NOT NULL,
+  `gasName` varchar(100) NOT NULL,
+  `reading` double NOT NULL,
+  `unitOfReading` varchar(100) DEFAULT NULL,
+  `latitude` double DEFAULT NULL,
+  `longitude` double DEFAULT NULL,
+  `sensorType` varchar(100) NOT NULL DEFAULT '',
+  `ro` double DEFAULT NULL,
+  `relHumidity` double DEFAULT NULL,
+  `tempInCelsius` double DEFAULT NULL,
+  `input` double DEFAULT NULL,
+  `resolution` double DEFAULT NULL,
+  PRIMARY KEY (`instant`,`deviceId`,`gasName`,`sensorType`),
+  KEY `sensorTypeIndex` (`sensorType`),
```

Comment:
I'd prefer to use the synonym `INDEX` because it's more obvious than `KEY`.

Reply:
great feedback, but we don't need to be that picky. one doesn't even see these index names in the code. But I'll remember for future tables such as the global constants table. Also the key thing is generated by the mysql automatically one should know this and you do! it's a pain to go back and change the key to index every time I use the tool to show the create table syntax.

Reply:
The index names are not used in the code. You can change the create file without changing the code.

Reply:
Philip there is a sweet balance between things that matter (e.g. functionality is affected) and things that are nice to have. This isn't critical by a long shot and it's eating up valuable time which we don't have.

Reply:
Naming and consistency affects maintainability which saves time in the long run. That's why it's a best practice.